### PR TITLE
DOS-1485: Add Rename Integration Tests 

### PR DESCRIPTION
**Pull request by peterc-intuit**
# What Changed

Added 6 additional test cases to test rename directive.

# Why

To test rename directive.

Todo:

- [X] Add tests
- [ ] Add docs

### CODE REVIEW

File: src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy [hunk 1]
```diff
@@ -92,4 +99,234 @@ class XtextConflictResolverSpec extends BaseIntegrationTestSpecification {
         queryType.getFieldDefinition("bar") != null
         queryType.getFieldDefinition("foo").type == queryType.getFieldDefinition("bar").type
     }
+
+    def "Query Service A"() {
+        given:
+        SimpleMockServiceProvider serviceA = createMockService("SVC_A",
```
Comment (ashpak-shaikh) [inline src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy:105]: Should we use inline schemas @CNAChino @kmoore-intuit ?

Comment (intoitz) [reply src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy:105]: The schemas are not too long and having it in localized places would enhance readability.

Comment (ashpak-shaikh) [reply src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy:105]: lets do inline schemas if that is a possibility

Comment (intoitz) [reply src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy:105]: Would it be also okay to inline the GraphQL queries?

Comment (CNAChino) [reply src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy:105]: I'm leaning on short schemas and short queries and have them all in the groovy file

Comment (kmoore-intuit) [reply src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy:105]: We could have the schema queries/response in a separate file for each spec and just import the variable in.  Would help keep the spec files clean and help with consistency.

Comment (peterc-intuit) [reply src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy:105]: Would it be possible to meet to come to an agreement?

Comment (peterc-intuit) [reply src/test/groovy/com/intuit/graphql/orchestrator/integration/XtextConflictResolverSpec.groovy]: Per discussion, the schemas and responses have been incorporated into respective unit test cases.

**Pull request merged**